src/loaders/revealjs-loader.js

```diff
@@ -48,6 +48,16 @@ var SCRIPT_END_PLACEHOLDER = "__SCRIPT_END__";
 function slidify(markdown, options) {
   options = getSlidifyOptions(options);
 
+  marked.use({
+    renderer: {
+      paragraph(text) {
+        const elClass =
+          getEmbeddedClasses(text, DEFAULT_ELEMENT_ATTRIBUTES_SEPARATOR) || "";
+        return `<p ${elClass}>${text}</p>`;
+      },
+    },
+  });
```

Comment:
This looks like it has a global side-effect on marked, which I'd rather avoid. Is there a version of this that does not have this downside?

Reply:
Yes you right ! What I do not understand is that feature already exist inside reveal.js markdown https://revealjs.com/markdown/ but it actually do not works.

Reply:
To avoid the global effect you may use the renderer option at the marked call site (at the end of the createMarkdownSlide function).

Sensei uses a modified copy of the Reveal markdown plugin so it might work a bit differently, so it can be normal that this feature does not work right now.

Reply:
Seems like I was wrong, passing the `renderer` option to the `marked` function is different from using `marked.use` as it will completely replace the renderer instead of extending it. `marked.use` seems to be the recommended way to do this. I find it very weird but we might aswell ascribe to the lib's documentation.